frontend/src/components/Uploader/ServiceAreasUploader.tsx

```diff
@@ -60,15 +61,17 @@ function getCounties(serviceAreas: [string, string][]): string[] {
  * Aliases are compared to column names as lowerCase()
  */
 const COLUMNS = [
-  { aliases: ['CountyName', 'County Name', 'County'] },
+  { aliases: ['CountyName', 'County Name', 'County'], required: true },
   { aliases: ['ZipCode', 'Zip Code', 'Zip'] }
 ]
 
 let parse = parseRows<[string, string][], { state: State }>(COLUMNS, (([county, zip], rowIndex, { state }) => {
 
+  // Note - we currently use '00000' as a placeholder in all counties.
+  zip = '00000'
   // Validate that zip exists
   if (zip && !(zip in COUNTIES_BY_ZIP)) {
```

Comment:
is this necessary at all if we're not using the input zip?

Reply:
Removed it and then added it again. I think we are likely to come back to zips, and it could also help debug codegen issues our db issues I think.